swift/Tests/Keystore/KeystoreKeyTests.swift

```diff
@@ -119,4 +119,25 @@ class KeystoreKeyTests: XCTestCase {
         let data = keystore.decryptPrivateKey(password: password)
         XCTAssertEqual(data?.hexString, "4357b2f9a6150ba969bc52f01c98cce5313595fe49f2d08303759c73e5c7a46c")
     }
+
+    struct KdfParams: Decodable {
+        let dklen: Int
+        let n: Int
+    }
+
+    struct EncryptionParameters: Decodable {
```

Comment:
can we have ability to specify `EncryptionParameters` on wallet creation?

Reply:
Done, see below